src/masternodes/anchors.cpp

```diff
@@ -354,7 +355,30 @@ void CAnchorAuthIndex::PruneOlderThan(THeight height)
 }
 
 CAnchorIndex::CAnchorIndex(size_t nCacheSize, bool fMemory, bool fWipe)
-    : db(new CDBWrapper(GetDataDir() / "anchors", nCacheSize, fMemory, fWipe))
+    : db(new CDBWrapper(GetDataDir() / "anchors", nCacheSize, fMemory, fWipe)),
+      PendingOrder([](const CAnchorIndex::AnchorRec& a, const CAnchorIndex::AnchorRec& b)
```

Comment:
It can be a free function

Reply:
Thought that was not going to work, but it does. Check it out.

https://github.com/DeFiCh/ain/pull/314/commits/ec77e3d48055d71f716d3e111199f510505e1e05